Makefile

```diff
@@ -26,10 +26,8 @@ capnpc-java : $(CAPNPC_JAVA_SOURCES)
 	$(CXX) $(CAPNPC_JAVA_SOURCES) $(CXX_FLAGS) -o capnpc-java
 
 install:
-	mkdir -p ${PREFIX}/bin
-	mkdir -p ${PREFIX}/include/capnp/
-	cp capnpc-java ${PREFIX}/bin
-	cp compiler/src/main/schema/capnp/java.capnp ${PREFIX}/include/capnp/
+	install -D capnpc-java "${DESTDIR}/${PREFIX}"/bin/capnpc-java
```

Comment:
What if `PREFIX` is a relative path?

Reply:
Is this about PREFIX not starting with `/`? This would be uncommon, although I believe the current change would accept it and still do the right thing™, because there is an explicit `/` between `DESTDIR` and `PREFIX`.

Reply:
If `PREFIX` is a relative path and `DESTDIR` is empty, then the `/` will make the result an incorrect absolute path.

Reply:
If you are saying that I should remove the explicit `/` between `DESTDIR` and `PREFIX`, then I am happy to do so. :)